model-metadata/model-metadata.md

```diff
@@ -4,35 +4,46 @@ This document describes the structure and content of a top-level Model Metadata
-| `model_id`           | string             | **REQUIRED.** A unique string identifier for the model. This ID must be unique across the provider. |
-| `algorithm_type`     | [Algorithm Type]\|string | **REQUIRED.** A string identifying the high-level algorithm type that the model employs. STRONGLY RECOMMENDED to use on the standard [Algorithm Type] values, but other values are allowed. |
-| `model_type`         | [Model Type]\|string | **REQUIRED.** Identifier for the type of model. STRONGLY RECOMMENDED to use on of the standard [Model Type] values, but other values are allowed. |
-| `model_architecture` | string             | Identifies the model architecture used (e.g. RCNN, U-Net, etc.).                    |
-| `license`            | string             | **REQUIRED.** The model's license(s). Either a SPDX [License identifier], `various` if multiple licenses apply, or `proprietary` for all other cases. |
-| `authors`            | \[[Author]\]       | **REQUIRED.** List of names and contact information for the model author(s).                 |
-| `citation`           | [Citation]         | Citation information related to the model.                                          |
-| `training`           | [Model Training]   | **REQUIRED.** A description of the data and environment used to train the model.    |
-| `inputs`             | \[[Model Input]\]  | **REQUIRED.** A list of [Model Input] objects describing the names and types of model inputs.  |
-| `outputs`            | \[[Model Output]\] | **REQUIRED.** A list of [Model Output] objects describing the names and types of model outputs. |
-| `runtimes`           | \[[Runtime]\]      | A list of [Runtime] objects describing serialized or containerized versions of the model that can be used to generate inferences. |
-| `usage_recommendations` | [Usage Recommendations] | A description of the recommended conditions under which the model can be used. |
-
-### Algorithm Type
-
-This string describes the general type of machine learning algorithm employed by the model. It is STRONGLY RECOMMENDED 
-that you use one of the following values, but other values are allowed.
+| Field Name              | Type                    | Description                                                                                                                                           |
+| ----------------------- | ----------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `model_id`              | string                  | **REQUIRED.** A unique string identifier for the model. This ID must be unique across the provider.                                                   |
+| `model_type`            | [Model Type]            | **REQUIRED.** Identifier for the type of model. STRONGLY RECOMMENDED to use on of the standard [Model Type] values, but other values are allowed.     |
```

Comment:
"on" --> "one"

I tried editing this inline but don't know where that went.

Reply:
Yeah, my markdown auto-formatter made a bunch of changes that I didn't notice prior to committing. I'll make that change.

Reply:
That description was out-of-date anyway, so I updated it in 35ca865d3b608f5ce8208647e47e588fca25252c